packages/block-library/src/image/save.js

```diff
@@ -73,9 +75,5 @@ export default function save( { attributes } ) {
 		);
 	}
 
-	return (
-		<figure { ...useBlockProps.save( { className: classes } ) }>
-			{ figure }
-		</figure>
-	);
+	return <figure { ...blockProps }>{ figure }</figure>;
```

Comment:
Are the changes here needed?

Reply:
Not anymore ... I think they are hangovers from when we were trying to wrap the images in a `<ul>` - reverted now.